src/site/content/en/index.njk

```diff
@@ -141,6 +185,23 @@ date: 2018-11-05
     </div>
   </section>
 
+  <script>
```

Comment:
I'd probably have a bunch of nits (e.g., `<script type="module">`) if it wasn't just for one day but this is fine :)